manageiq-operator/api/v1alpha1/helpers/miq-components/kafka.go

```diff
@@ -253,6 +253,10 @@ func KafkaDeployment(cr *miqv1alpha1.ManageIQ, scheme *runtime.Scheme) (*appsv1.
 				Name:  "ALLOW_PLAINTEXT_LISTENER",
 				Value: "yes",
 			},
+			corev1.EnvVar{
+				Name:  "KAFKA_CFG_ADVERTISED_LISTENERS",
+				Value: "PLAINTEXT://kafka:9092",
```

Comment:
This is fine for the initial PR, but we should set up authentication and optional SSL/TLS in a followup